slapo/model_dialect/deepspeed/pipeline.py

```diff
@@ -293,3 +294,25 @@ def forward(self, *args, **kwargs):
         )
         ret = tuple(ret)
         return ret
+
+
+@register_model_dialect("deepspeed", "pipeline_engine")
+def deepspeed_pipe_engine(
+    stage_modules,
+    topology,
+    param_dtype,
+    **kwargs,
+):
+    from deepspeed import pipe
+
+    model = pipe.PipelineModule(
+        stage_modules,
```

Comment:
Is `stage_modules` a list of layers? Or is it partitioned by the Slapo schedule?